pulpcore/tests/functional/api/test_crud_repos.py

```diff
@@ -34,11 +34,17 @@ def test_01_create_repo(self):
     def test_02_create_same_name(self):
         """Try to create a second repository with an identical name.
 
-        See: `Pulp Smash #1055
+        * `Pulp Smash #882 <https://github.com/PulpQE/pulp-smash/issues/882>`_.
+        * `Pulp Smash #1055
         <https://github.com/PulpQE/pulp-smash/issues/1055>`_.
         """
-        with self.assertRaises(HTTPError):
-            self.client.post(REPO_PATH, gen_repo(name=self.repo['name']))
+        self.client.response_handler = api.echo_handler
+        response = self.client.post(
```

Comment:
I suggest 

```py
reponse = api.Client(self.cfg, api.echo_handler).post(...)
```
instead of mutating the existing client handler. (or moving the handler back to its original handler after the test)

Reply:
@rochacbruno, thanks for your review.  I did not update the client to the original `response_handler` since a new client is create for each test.

```python
    def setUp(self):
        """Create an API client."""
        self.client = api.Client(self.cfg, api.json_handler)
```